src/main/scala/DocGenerator.scala

```diff
@@ -245,7 +251,45 @@ object LatexDocGenerator extends DocGenerator {
                     """
   }
 
+
+  def hypertarget(e: {def id: Seq[Tree]}, text: Option[String]): String = {
+    val t = text.getOrElse("")
+    if (e.id.isEmpty)
+      t
+    else
+      s"\\hypertarget{${link(e.id)}}{$t}"
+  }
+
+  def hyperlink(e: {def id: Seq[Tree]}, text: Option[String]) = {
+    val t = text.getOrElse("")
+    if (e.id.isEmpty)
+      t
+    else
+      s"\\hyperlink{${link(e.id)}}{$t}"
+  }
+
+  def link(tpe: Seq[Tree]) = {
+    val termToTerm = "."
+    val termToType = "."
+    val typeToType = "#"
+    val typeToTerm = "_"
+    def separtor(s: Tree, i: Int): String = if (s != tpe.last) {
```

Comment:
If you want reference equality, go for `eq`. If you want comparison that takes into account name resolution, go for `==` (but at the moment it will most likely not work).